### ISSUE TITLE
Module: grubcfg - have_distributor_line error on installation

### ISSUE THREAD

**Issue by arcolinuxz** [calamares/calamares#1283]
Package calamares : 3.2.17 
Build here :  shorturl.at/gjpyF
I have not changed the configs (as we have 17 of them) - those contain the settings for calamares.
Build went fine.
Installation went fine but when it got to the grubcfg module it reported an error
![grubcfg](https://user-images.githubusercontent.com/36679406/69917932-fbfdec80-146b-11ea-9e0a-f8b9459aca45.jpg)
We do not supply a grubcfg.conf file in our settings at /etc/calamares/...
In other words we rely on the one in /usr/share/calamares/modules/grubcfg.conf.

Do we need to create a particular grufbcfg.conf file or is there a typo in the code?

**Comment by philmmanjaro** [comment]
already fixed 21 hours ago: https://github.com/calamares/calamares/commit/1a13704c71dda7d2656ee12fef3f58d9603027e5

**Comment by arcolinuxz** [comment]
great - that is the service we like from calamares - I let @adriaandegroot  or others close it

**Comment by philmmanjaro** [comment]
We will provide a new tarball with the fix and updated translations soon.

**Comment by adriaandegroot** [comment]
Released 3.2.17.1 just now.

**Comment by arcolinuxz** [comment]
just reporting that installation went fine with this release
